crates/trie/src/hashed_cursor/post_state.rs

```diff
@@ -210,13 +210,24 @@ pub struct HashedPostStateStorageCursor<'b, C> {
 }
 
 impl<'b, C> HashedPostStateStorageCursor<'b, C> {
-    fn was_storage_wiped(&self, account: &H256) -> bool {
+    fn is_db_storage_wiped(&self, account: &H256) -> bool {
         match self.post_state.storages.get(account) {
             Some(storage) => storage.wiped,
             None => false,
         }
     }
 
+    /// Check if the slot was zeroed out in the post state.
+    /// The database is not checked since we don't insert zero valued slots.
+    fn is_slot_empty(&self, account: &H256, slot: &H256) -> bool {
```

Comment:
this returns false if no entry exists, is this expected?

should this be called `is_(touched?)_slot_value_zero`?

Reply:
that is indeed expected. `is_touched_slot_value_zero` sounds much clearer